website/payments/views.py

```diff
@@ -144,15 +145,27 @@ def get_queryset(self) -> QuerySet:
         )
 
     def get_context_data(self, *args, **kwargs):
+        current_time = timezone.now().replace(tzinfo=None)
+        nr_months = 0
+        earliest_membership = self.request.member.earliest_membership
+        difference = relativedelta(current_time, earliest_membership.since)
```

Comment:
This crashes if a user with no memberships tries to open it:
![image](https://github.com/user-attachments/assets/dc762f7d-f8ee-4e40-847b-39e0fbfec57d)